exchanges/gateio/gateio_wrapper.go

```diff
@@ -2364,6 +2360,19 @@ func (g *Gateio) WebsocketSubmitOrder(ctx context.Context, s *order.Submit) (*or
 	}
 }
 
+func timeInForceString(tif order.TimeInForce) string {
+	switch {
+	case tif.Is(order.PostOnly):
+		return "poc"
+	case tif.Is(order.ImmediateOrCancel):
+		return order.ImmediateOrCancel.Lower()
+	case tif.Is(order.FillOrKill):
+		return order.FillOrKill.Lower()
```

Comment:
🚧 Can you add a comment to explain *why* we're using rules this function is following?

Specifically, I'm reading that if the tif is `POC`, `IOC` or `FOK` then we'll use the lowercase value, but if it's not, then we'll ... also use the lowercase value.

I'm *INFERRING* from that that we're saying "If it's one of the first 3, it might also be something else, but we're going to just use POC, IOC or FOK. If it's anything *else* then we'll use the compound value"
In this case, given it's an exchange api, maybe just list the explicit values that we're going to use, because re-using the GCT `order` package strings carries the risk of the two becoming disconnected.

Reply:
hardcoding the values sounds nice. However, It is just not about making it lowercase ... the TimeInForce instance could have post only and other TimeInForcevalues in it. added with the binary _or_  operator and we are making sure this three TimeInForce values are returned with out other TimeInForce value added to them separatem by comma.

example: For `order.GoodTillCancel | order.PostOnly` we want `poc` 
                For `order.ImmediateOrCancel | order.AnyOther` we want  `ioc` ... and forward the rest to the original function to handle

Reply:
* 🚧 Understood; But I think you misunderstood what I'm asking for:
> Can you add a comment
```suggestion
// timeInForceString returns the most relevant time-in-force exchange string for a TimeInForce
// Any TIF value that is combined with POC, IOC or FOK will just return that
// Otherwise the lowercase representation is returned
```

The point of a comment like this is to make sure that the *intention* is decoupled from the implementation. So if there was a bug, someone know what we're trying to achieve. And so they can refactor it whilst following the intentions.

* But I'm left wondering: Isn't the only remaining valid value `gctTIF` ?
So yeah, for the sake of simplicity and not leaving the reader wondering, maybe just put that in there.
And if you use a case for that, I wouldn't mind having a default that `panics`, since it should be unreachable.

Reply:
Moving to new thread for clear visibility again.